js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1039,6 +939,17 @@ const EditingController = modules.ViewController.inherit((function() {
             this._focusFirstEditableCellInRow(rowIndex);
         },
 
+        _checkFocus: function() {
+            return true;
+        },
+
+        _checkEditItem: function() {
+            return true;
+        },
+
+        _validateBeforeEditCell: noop,
```

Comment:
I think editing modules should not know about validation. So, all methods with the "validate" word should be moved to the validating module.

Reply:
I think it doesn't matter how are called methods which implementations are placed in extenders. I can rename it and according your point of view it will be OK, but the essence remains the same - editing calls some methods which are implemented in validating/KBN/adaptivity/elsewhere. IMO, we should think simpler - editing must not know about validating _controller_ or contain validating _implementation_.